### DISCUSSION TITLE
$data afterFormValidated in Tables\Actions\CreateAction not filled

### DISCUSSION THREAD

**Discussion by untillnesss** [filamentphp/filament#14945] [Help]
### Package

Panel builder

### Package Version

v3.2.127

### How can we help you?

Previously, when I filled the callback parameter in `afterFormValidated `which is located in `Tables\Actions\CreateAction` with `$data`, it would fill in data about the form.
![image](https://github.com/user-attachments/assets/17cd26d9-e9d9-47bb-827c-fffdbf77d17c)

But after I did some Filament version updates by running `composer update -W`, it now doesn't work at all.
![{BC6B1F78-85DE-4EC5-B4A3-977DD1CB7E5E}](https://github.com/user-attachments/assets/addbf050-32ec-466a-9138-bae3f7ffb27b)

The following is an example of the full coding.
[Click Here Gist](https://gist.github.com/untillnesss/4afe0b63d7814dfe8475c84a6e7192b4)

### Even though previously everything was running smoothly.

**Comment by leandrocfe** [comment]
Do you know the version that it was working?

Alternatively, I think you can inject `$livewire` and use `mountedTableActionsData`

```php
->afterFormValidated(function (RelationManager $livewire) {
    $data = $livewire->mountedTableActionsData[0];
})
```

**Reply by aurawindsurfing** [reply]
Its ugly but it works
